R/pkg/R/DataFrame.R

```diff
@@ -2766,18 +2780,21 @@ setMethod("histogram",
             return(histStats)
           })
 
-#' Saves the content of the SparkDataFrame to an external database table via JDBC
+#' Save the content of DataFrame to an external database table via JDBC.
 #'
-#' Additional JDBC database connection properties can be set (...)
+#' Saves the content of the SparkDataFrame to an external database table via JDBC. Additional JDBC
```

Comment:
I also think we should consociate the plurality of first word which is verbal in description